src/CoreDAL/UserListRepository.cs

```diff
@@ -47,17 +47,17 @@ public async Task<UserDto> UpdateById(Guid id, UserDto user)
             return await Task.FromResult((UserDto)null);
         }
 
-        public async Task<Guid?> RemoveById(Guid id)
+        public async Task<int> RemoveById(Guid id)
```

Comment:
In general you can remove this repository

Reply:
it's for future testing to avoid using InMemory DB

Reply:
We will use mocks and only interfaces instead of using explicit implementation.

Reply:
ok, removed